ui/src/main/java/org/mifos/mobile/core/ui/theme/Theme.kt

```diff
@@ -7,15 +7,15 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.LocalContext
 
 private val LightThemeColors = lightColorScheme(
```

Comment:
See example of dark and light theme implementation https://github.com/android/nowinandroid/blob/main/core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/theme/Theme.kt

and directly picking the color from theme instead of mentioning the static color in composable